src/fontra/views/editor/editor.html

```diff
@@ -32,6 +32,13 @@
       --sidebar-tab-width: 3em;
       --sidebar-content-width-left: 20em;
       --sidebar-content-width-right: 22em;
+      --resize-cursor: col-resize;
+    }
+
+    :root.resizing {
```

Comment:
Looking at it now I think this class should be called "sidebar-resizing", to make it more obvious what's goin on here. I also wonder if we can move this entire `:root` block closer to the sidebar CSS, for the same reason.

Reply:
I think class "sidebar-resizing" is ok, more explicit is the better. I will send a patch.

What do you mean by moving entire `:root` to the sidebar css?

Reply:
Lowering by one block? There is only sidebar container between them.

Reply:
Ah yes, lowering by one block is good, thanks.

Reply:
Sent a patch.